game/view/gameplay/actionhud/init.lua

```diff
@@ -109,15 +105,29 @@ function ActionHUD:init(route)
   ADJACENCY.unset(self.adjacency)
 
   -- Control hints
-  self.cancel_hint = CancelHint(13, 8)
+  self.hand_hint = ControlHint(240+13, 28, "button-key-draw_hand", {
+    DEFAULT = "Open hand", FOCUS = "Close hand"
+  })
+  self.hand_hint:register("HUD_MIDDLE")
+  self.open_packs_hint = ControlHint(240+138, 8, "button-key-open_packs", {
+    DEFAULT = "See packs", FOCUS = "Discard"
```

Comment:
ok i know this would require massive amounts of refactoring but... a thought: maybe visual text should be stored somewhere using unique keys (aka LocaleKeys)? this way you can organize ingame-flavour text and even expand into translation in the far future. im pretty sure this is done with the core attributes of the game in a way, isn't it?

Reply:
Yeah... maybe someday. (#1464)